.travis.yml

```diff
@@ -72,9 +85,8 @@ matrix:
         - lcov --capture --directory "$TRAVIS_BUILD_DIR/build" --output-file coverage.info
         - lcov --remove coverage.info '/usr/*' "$LOCAL_INSTALL/*" --output-file coverage.info # filter system-files and local-dependencies
         - lcov --list coverage.info # debug info
-        - bash <(curl -s https://codecov.io/bash) -f coverage.info || echo "Codecov did not collect coverage reports"
-
-
+        - bash <(curl -s https://codecov.io/bash) -f coverage.info || echo "Codecov did not collect coverage reports"      
+  
```

Comment:
Nothing should be changed here

Reply:
Rolled back in https://github.com/precice/precice/pull/421/commits/7c2ec4b1f06467d37244bf51177e0eccb134d453